src/docs/development/data-and-backend/state-mgmt/options.md

```diff
@@ -35,6 +35,14 @@ A recommended approach.
 * [You might not need Redux: The Flutter edition][], by Ryan Edge
 * [Making sense of all those Flutter Providers][]
 
+## Riverpod
+
+A provider rewrite by the same author, compiled safe, more testable, more flexible
```

Comment:
This reads like an advert. Please rewrite so that:

1. This can stand on its own (not "rewrite of the above") — people scan these pages, and the order of the items can change at any time.
2. The sentence or two distill the essence of the package. 

So maybe something like:

> An approach similar to Provider that is compile-safe and testable. It doesn't have a dependency on the Flutter SDK.

Also, nit: There should be a period at the end of the sentence, for consistency.